File: Edabit/java/one.java
/*

Create a method that takes an integer as an argument and returns
"even" for even integers and "odd" for odd integers.

Run using:
  java one.java

*/

public class Program
{
  // internal function
  public static String isEvenOrOdd(int num)
  {
    return ((num % 2) == 0) ? "even" : "odd";
  }
  // main function
  public static void main(String[] args)
  {
    System.out.printf("%s\n", isEvenOrOdd(3));
  }
}

File: Edabit/php/one.php
<?php

/*
create a function that takes a number as its only argument and returns true if it's less than
or equal to zero, otherwise return false.
*/

function lessThanOrEqualToZero($num)
{
  if ($num <= 0)
    return true;
  return false;
}

$r = lessThanOrEqualToZero($argv[0]);

var_dump( $r );

?>
